m3/src/main/java/com/uber/m3/tally/m3/M3Sanitizer.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package com.uber.m3.tally.m3;
+
+import com.uber.m3.tally.sanitizers.ScopeSanitizer;
+import com.uber.m3.tally.sanitizers.ScopeSanitizerBuilder;
+import com.uber.m3.tally.sanitizers.ValidCharacters;
+
+public class M3Sanitizer {
+
+    /**
+     * Creates the default M3 sanitizer.
+     * @return default M3 sanitizer
+     */
+    public static ScopeSanitizer create() {
+        return new ScopeSanitizerBuilder()
+            .withReplacementCharacter(ValidCharacters.DEFAULT_REPLACEMENT_CHARACTER)
+            .withNameCharacters(
+                ValidCharacters.of(
+                    ValidCharacters.ALPHANUMERIC_RANGE,
+                    ValidCharacters.UNDERSCORE_DASH_DOT_CHARACTERS))
+            .withKeyCharacters(
+                ValidCharacters.of(
+                    ValidCharacters.ALPHANUMERIC_RANGE,
+                    ValidCharacters.UNDERSCORE_DASH_CHARACTERS))
```

Comment:
iirc dot is also a valid character but not recommended

Reply:
changed to UNDERSCORE_DASH_CHARACTERS